pkgs/development/python-modules/rns/default.nix

```
@@ -5,6 +5,7 @@
   fetchFromGitHub,
   netifaces,
   pyserial,
+  python3,
```

```suggestion
  python,
```
because this has to use Python 3.11 if we're using `python311.pkgs.rns`.